File: modules/monkeytools/native/MonkeyTools.cs
// MonkeyTools.cs

class MonkeyTools
{
	public static void ExitApp()
	{
        gxtkApp.game.Exit();   // exit app
    }
	public static void ShowMouse()
	{
		gxtkApp.game.IsMouseVisible=true;
	}
	public static void HideMouse()
	{
		gxtkApp.game.IsMouseVisible=false;
	}
}

File: modules/monkeytools/native/MonkeyTools.cpp
// MonkeyTools.cpp

class MonkeyTools
{
public:
	static void ExitApp()
	{
		exit(0);	// exit app
	}
	static void ShowMouse()
	{
		ShowCursor(true);
	}
	static void HideMouse()
	{
		ShowCursor(false);
	}
};
